jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/builders/NodeListXYBuilder.java

```diff
@@ -0,0 +1,60 @@
+package us.dot.its.jpo.ode.plugin.j2735.builders;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import us.dot.its.jpo.ode.plugin.j2735.J2735ComputedLane;
+import us.dot.its.jpo.ode.plugin.j2735.J2735NodeListXY;
+import us.dot.its.jpo.ode.plugin.j2735.J2735NodeXY;
+
+public class NodeListXYBuilder {
```

Comment:
It is possible to annotate the classes with SEQUENCE-OF (array) properties with XML-specific annotations, something like this:

```java
public class J2735NodeListXY extends Asn1Object {
	
	@JacksonXmlElementWrapper(localName = "nodes")
	@JacksonXmlProperty(localName = "NodeXY")
	private J2735NodeXY[] nodes;
	
    ...
```

That would tell Jackson to read the XML with the <nodes><NodeXY> wrapper that it expects, but is ignored when writing JSON, so the JSON wouldn't be wrapped.   Then we wouldn't need Builder classes like this to manually handle the arrays at all.